com.gitlab.adnan338.Invoicer.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.gitlab.adnan338.Invoicer
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+command: invoicer
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=xdg-data:rw
```

Comment:
What does it uses this for?

Reply:
The app uses `~/.local/share` on posix systems to store persistent data. It serializes/deserializes the data from/for its data model.

Reply:
You have access to that directory by default; except it's a sandboxed one at ~/.var/app/$app-id; just make sure you're not hardcoding the path to ~/.local/share

Reply:
How may I get that directory from the app. Can I just use hardcoded ` ~/.var/app/com.gitlab.adnan338.Invoicer`?

Reply:
glib has a helper function `g_get_user_data_dir`, it just reads `$XDG_DATA_HOME`

Reply:
Also the app has write access to `$XDG_DATA_HOME` by default.

Reply:
Ok [the latest commit](https://github.com/flathub/flathub/pull/2054/commits/4f913523b885da2d016cbe5b7155f0bc19eadb2c) addresses this.